test/integration/adapters.js

```diff
@@ -192,7 +171,7 @@ describe('Adapters integration', function () {
       })
 
       describe('Suite with only one passing test', function () {
-        _testSuite(3, 's')
+        _testSuite(3, _testSimpleSuite.bind(null))
```

Comment:
Why do you need the `bind` call? If you really need a specific context, use `call` or `apply` at the call site, like `testSpecificSuite.call(null, index)`